### ISSUE TITLE
Battery icon

### ISSUE THREAD

**Issue by mamo-nick** [atc1441/ATC_MiThermometer#218]
Display shows battery level 99 %, but battery icon looks like empty battery.
Is it bug or feature?

**Comment by atc1441** [comment]
The battery symbol indicates that the shown value is the battery value. 

It is 99% then.

If the battery symbol is off the shown value is the humidity

**Comment by mamo-nick** [comment]
Yes, I know it,  but I thought about baterry. E.g. when battery level is 90 %. icon wil be mdi:battery-90 and when 20 %, battery-20. In both case, display shows battery-20.

**Comment by atc1441** [comment]
Please show a picture of what you mean
